apps/aecore/lib/aecore/wallet/worker.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Aecore.Wallet.Worker do
+  @defmodule """
+  Module for handling the creation of a Wallet file
+  """
+
+  use GenServer
+
+  @typedoc "Options for network"
+  @type opts :: :mainnet | :testnet
+
+  @aewallet_dir Application.get_env(:aecore, :aewallet)[:path]
+  @aewallet_pass "1234"
```

Comment:
why do we choose "1234" as password over "" as default?

Reply:
As I mentioned up, I'll fix this